### ISSUE TITLE
Configure NextCloud based off policy discussion

### ISSUE THREAD

**Issue by justwheel** [RITlug/config-management#8]
Make adjustments and build prototype of NextCloud server to offer to RITlug's community

* **Blocked by**: _policy discussion_
* **DUE**: TBD

**Comment by justwheel** [comment]
@ct-martin To follow up, is the Nextcloud installation still live in our infrastructure? Where did we leave off on this issue?

**Comment by Tjzabel** [comment]
This issue is no longer blocked on anyone.

@ct-martin is now free to run wild :horse:

**Comment by Tjzabel** [comment]
I am going to close this issue for now as out of scope.
